contracts/ETHHelper.sol

```diff
@@ -5,7 +5,7 @@ import "@openzeppelin/contracts/access/Ownable.sol";
 import "@ironblocks/firewall-consumer/contracts/FirewallConsumer.sol";
 
 contract ETHHelper is Ownable, FirewallConsumer {
-    constructor() {
+    constructor() Ownable(_msgSender()) {
```

Comment:
https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4267
![image](https://github.com/The-Poolz/Poolz-Helper/assets/68740472/6f9c91c4-e8f6-4a97-a51e-fcca60c2967f)